lib/async/http/protocol/http1/server.rb

```diff
@@ -50,10 +50,7 @@ def each(task: Task.current)
 							response = yield(request, self)
 							body = response&.body
 							
-							if @stream.nil? and body.nil?
-								# Full hijack.
-								return
-							end
+							return if hijacked?
```

Comment:
```suggestion
							if hijacked?
								body&.close
								return
							end
```

I wonder if we need this, just in case a body was returned.

Reply:
Let me check this carefully. IIRC ActionCable (the reason this PR was started in the first place) used to return something for the body, need to see if closing it is safe (e.g. if it's `ActionCable::Connection::Stream` then it's going to break things again).